cdap-metadata-spi/src/test/java/io/cdap/cdap/spi/metadata/MetadataStorageTest.java

```diff
@@ -1535,6 +1535,40 @@ private void testSortedSearch(MetadataStorage mds, Sorting sorting, List<Metadat
     Assert.assertNull(response.getCursor());
   }
 
+  @Test
+  public void testOneRequiredTerm() throws IOException{
+    MetadataStorage mds = getMetadataStorage();
+
+    String ns1 = "ns1";
+//    String ns2 = "ns2";
+//    MetadataEntity app1 = ofApp(ns1, "app1");
+//    MetadataEntity app2 = ofApp(ns2, "app1");
+//    MetadataEntity program1 = ofWorker(app1, "wk1");
```

Comment:
lets not have commented code in the test

Reply:
got it!